content/guides/models/app/features/panels/_index.md

```diff
@@ -102,6 +103,17 @@ To remove all panels from a manual workspace, click its action `...` menu, then
 
 To remove all panels from an automatic or manual workspace, you can [reset the workspace](#reset-a-workspace). Select **Automatic** to start with the default set of panels, or select **Manual** to start with an empty workspace with no panels.
 
+### Show specific panels to members of your organization
+Direct colleagues to a specific panel in your project. The link redirects users to a full screen view of that panel when they click that link. To create a link to a panel:
+
+1. Hover your mouse over the panel.
+2. Select the action `...` menu.
+3. Click **Copy panel URL**.
+
+The settings of the project determine who can view the panel. This means that if the project is private, only members of the project can view the panel. If the project is public, anyone with the link can view the panel.
+
+If you have multiple panels with the same name, W&B directs users to the first panel with that name.
```

Comment:
```suggestion
If multiple panels have the same name, W&B shares the first panel with the name.
```
The first according to what? Maybe we should advise not to get into this situation.